plugins/node/opentelemetry-instrumentation-redis-4/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@opentelemetry/instrumentation-redis-4",
+  "version": "0.29.0",
+  "description": "OpenTelemetry redis automatic instrumentation package.",
```

Comment:
Should we indicate in the description it is specific to version 4?

Reply:
added